pkg/model/op.go

```diff
@@ -124,19 +126,23 @@ func GetOpTypeAndResourceNameFromOpID(
 		}
 		resName := strings.TrimPrefix(opID, "Get")
 		if pluralize.IsPlural(resName) {
-			if resourceExistsInConfig(resName, cfg) {
-				return OpTypeGet, resName
+			// "pluralized singular" resources like EC2's DhcpOptions
+			// are explicitly defined in generator config to NOT singularize
+			// because there is no such thing as DhcpOption
+			if !resourceExistsInConfig(resName, cfg) {
+				return OpTypeList, pluralize.Singular(resName)
```

Comment:
why are we returning DhcpOption here, then?

Reply:
That's only the case if `DHCPOptions:` is not defined in the config. If it is, then `return OpTypeGet, resName` is hit on line 136 without singularizing the `resName`